AudioTrack/app/src/main/java/github/nisrulz/audiotrack/MainActivity.java

```diff
@@ -0,0 +1,119 @@
+package github.nisrulz.audiotrack;
+
+import android.media.AudioFormat;
+import android.media.AudioManager;
+import android.media.AudioTrack;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+// Contributed by Sergio Morales (@Fireblend)
```

Comment:
Since git system attributes the author to each line of code, adding this is not required. A simple git blame enlists all the information related to the commit/line of code.

Reply:
Will remove line.